src/shared/components/Banners/Banner/Banner.tsx

```diff
@@ -13,8 +13,8 @@ export const Banner = ({
   duration = 5000,
   onClose,
   hasCloseButton = !!onClose,
-  severity,
-  'data-testid': dataTestid,
+  severity = 'success',
+  'data-testid': dataTestid = `${severity}-banner`,
```

Comment:
I'm wondering if we could simplify the component by removing the 'data-testid' prop and according type for it and using a variable dataTestid instead. What are your thoughts on this change?

Reply:
I thought that it would be prudent to leave the ability to override the default value for the test ID if there was a need to uniquely target a specific banner. Also just confirming the requirements with task author based on [this issue](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1507#discussion_r1497231018) being raised, will report back!

Reply:
Sounds like Oleg is happy to stick with restrictive `data-testid`s as per the task requirements, so I'll remove this extra prop and simplify the code a bit as you suggested.